extensions/attention-stream/src/background.ts

```diff
@@ -409,31 +411,47 @@ async function stateChangeCallback(newState) {
             }
 
             // Pixels may be either HTTP GET requests for an image, or a POST from JS.
-            // If a POST is detected, collect the form data submitted as well.
+            // If a POST is detected, collect the form data
+            // or raw request body submitted as well.
             let formData;
+            let rawRequestBody;
             if (details.method === "POST") {
               const rawFormData = details.requestBody?.formData;
+              const rawRequestData = details.requestBody?.raw;
               if (rawFormData) {
                 formData = new URLSearchParams(rawFormData).toString();
               }
+              if (rawRequestData) {
```

Comment:
Does this mean we will unconditionally collect both the parsed form and also the raw data, if the tracker uses a multipart/form?

Reply:
I can imagine a rationale for collecting both but I'm curious what your thoughts here are - it seems like if an already-parsed form is available we should prefer that, and only get the raw request data if it isn't.

Reply:
yeah I was erring on the side of "collect redundant data vs. potentially miss data". It looks like the spec is that if the data is parsed into form data then the raw is empty anyway?

Reply:
Oh interesting, does the raw body get consumed if there's a form? If so I guess it's a moot point.